src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -80,6 +80,76 @@ public function prototype($type)
     }
 
     /**
+     * Shorthand for prototype('variable').
+     *
+     * @return VariableNodeDefinition
+     */
+    public function prototypeVariable()
+    {
+        return $this->prototype('variable');
+    }
+
+    /**
+     * Shorthand for prototype('scalar').
+     *
+     * @return ScalarNodeDefinition
+     */
+    public function prototypeScalar()
```

Comment:
`scalarPrototype` sounds better for me. The only motivation behind the current naming is to *group* the methods in an IDE auto complete list.